### ISSUE TITLE
Column command usage

### ISSUE THREAD

**Issue by jwadolowski** [Cognifide/CQ-Unix-Toolkit#28]
Hey,

I was wondering what was the reasoning behind `column` command flags used in the following tools:
- `cqapi`
- `cqjcr`
- `cqls`
- `cqosgi`
- `cqwfl`

Every single time the following flags/params are passed on the the `column` utility:
- `-x` - manpage says "fill columns before filling rows" but it seems to be totally ignored when consuming `stdin` output. Some example can be found [here](http://unix.stackexchange.com/questions/94536/fill-columns-before-rows-option-in-unix-column-command)
- `-t`  - no remarks at all as it is necessary. It also seems to defeat the purpose of `-x` and `-c X` params
- `-c X` - manpage says "output is formatted for a display columns wide", so if I understand that correctly in other words it means "align the output to a display that is X column (character) wide". Sometimes as `X` you use rather low number, i.e. `6` in `cqls`, so it does not make any sense to me.
- `-s X` - definition of a separator character, used properly in my opinion

Could you please explain it a little bit better? I tested CQ Unix Toolkit just on Linux boxes and some of these flags seems to be redundant (didn't test modified version thoroughly though). On the other hand it might be quite crucial for other platforms.

Thanks,
Kuba

**Comment by kitarek** [comment]
Yes - there are problems running column in FreeBSD. There some differences but I'm still not sure if we need all the switches. I would like to implement pretty and machine printing in the `cqapi` and replace all other implementation by single `cqapi` call and thus reducing implementation.

In this case we can check if it works properly on different *nix systems with minimal subset of flags.
